lib/awful/prompt.lua

```diff
@@ -64,6 +64,42 @@ local function history_check_load(id, max)
     end
 end
 
+local function is_word_char(c)
```

Comment:
This should be moved to `awful.util` - and there might be something like that maybe already..